collection/10.17028/rd.lboro.4665448.v1/SIPmetadata/SIP-1.request.json

```diff
@@ -47,8 +47,8 @@
     "objectDescription": "The fileset contains simulation files of the non-uniform meshed patch antennas (using FDTD Empire XCcel). The conductor thickness and conductivity can be adjusted according to the conductive threads.",
```

Comment:
Not sure this description is accurate for this SIP, as only one file is being included (a pdf)